README.md

```diff
@@ -1,5 +1,6 @@
 ## Welcome
-Welcome to the CloudBees examples repository. This repository is a place for storing tutorials, examples, and other learning materials. Of note:
+Welcome to the CloudBees examples repository. This repository is a place for storing tutorials, examples, and other learning materials to complement mentions in product documentation. 
+Of note:
 
 * It is owned and managed by the docs team. Please contact a docs team member or manager for assistance in adding files.
```

Comment:
I know you didn't write this part Patty, but this seems odd because it is a public facing repo, so anyone outside of CloudBees can't contact the docs team.

Reply:
```suggestion
* It is owned and managed by the docs team.
```